lib/workers/pr/changelog/source-github.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
 async function getTags(endpoint, versionScheme, repository) {
   let url = endpoint
     ? endpoint.replace(/\/?$/, '/')
-    : 'https://api.github.com/';
+    : /* istanbul ignore next: not possible to test, maybe never possible? */ 'https://api.github.com/';
```

Comment:
@rarkins I think endpint can never be null or undefined here. Could not create a test for this.